vortexasdk/endpoints/products.py

```diff
@@ -68,19 +68,16 @@ def search(
         [VortexaAPI Product Reference](https://docs.vortexa.com/reference/POST/reference/products)
 
         """
-        search_params = {
+        api_params = {
             "term": convert_to_list(term),
             "ids": convert_to_list(ids),
             "product_parent": convert_to_list(product_parent),
             "allowTopLevelProducts": True,
         }
 
-        search_result = super().search(**search_params)
-
-        if exact_term_match:
-            return ProductResult(filter_exact_match(term, search_result))
-        else:
-            return ProductResult(search_result)
+        return ProductResult(
```

Comment:
this logic now exists in the `Search` class, so that Vessels, Corporations, Geographies can use it without duplicating the code.